docs/project_learning_objectives.md

```diff
@@ -0,0 +1,33 @@
+# [Draft] Project Learning Objectives
+
+We have come up with a list of learning objectives for this project, but it's a dynamic ever growing list. If we have something we are wanting to learn currently, then we can add that as an objective and try and come with some work or feature that anchors on the learning objective. For now, we have added them as labels that can be added to tasks and PRs:
+
+---
+
+## MVP / First phase
```

Comment:
What I'm thinking is that we change this to "MVP – make it work" and then we remove a couple of the learning objectives from this phase such as "Object Oriented Learning", "Design & Architecture" and "Scalability". 

Then we add those learning objectives to the next phase after we have something working. What do you reckon?

Reply:
Yeah that makes it easy to compartmentalise the objectives and have a more concise set for each phase.

Reply:
Yeah that makes it easy to compartmentalise the objectives and have a more concise set for each phase.